### ISSUE TITLE
selfupdate subcommand is corrupted?

### ISSUE THREAD

**Issue by cosmo0920** [embulk/embulk#280]
I freshly installed latest Embulk(0.7.1) into Ubuntu machine with Embulk README instruction: https://github.com/embulk/embulk#linux--mac--bsd

And then I execute `embulk selfupdate` command:

### Expected

Embulk selfupdate subcommand success.

### Actual

```bash
% embulk selfupdate
2015-08-20 16:25:55.645 +0900: Embulk v0.7.1
ArgumentError: wrong number of arguments (0 for 1)
      run at /home/hhatake/.embulk/bin/embulk!/embulk/command/embulk_run.rb:256
    <top> at /home/hhatake/.embulk/bin/embulk!/embulk/command/embulk_main.rb:2
  require at org/jruby/RubyKernel.java:940
   (root) at uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1
    <top> at file:/home/hhatake/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:55
```

**Comment by frsyuki** [comment]
strange...

**Comment by cosmo0920** [comment]
I tried other version of Embulk.
I got different result 0.6.25 and 0.7.0.

### Embulk 0.6.25:

I got expected result.

```bash
% embulk selfupdate
2015-08-20 16:39:34.461 +0900: Embulk v0.6.25
Checking the latest version...
Found new version 0.7.1.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.7.1.jar ...
Updated to 0.7.1.
```

### Embulk 0.7.0

I got same error as above:

```bash
% embulk selfupdate
2015-08-20 16:40:56.142 +0900: Embulk v0.7.0
ArgumentError: wrong number of arguments (0 for 1)
      run at /home/hhatake/.embulk/bin/embulk!/embulk/command/embulk_run.rb:256
    <top> at /home/hhatake/.embulk/bin/embulk!/embulk/command/embulk_main.rb:2
  require at org/jruby/RubyKernel.java:940
   (root) at uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1
    <top> at file:/home/hhatake/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:55

```

**Comment by frsyuki** [comment]
I see. embulk_run.rb:256 is `Embulk.selfupdate(system)`. And the selfupdate method accepts 1 argument. So, I wonder why "ArgumentError: wrong number of arguments (0 for 1)" happens.

**Comment by cosmo0920** [comment]
I think this error is caused by argument's `system`.
This argument is interpreted built-in `system` function.

To clarify this, I try to modify like this:

```diff
diff --git a/lib/embulk/command/embulk_run.rb b/lib/embulk/command/embulk_run.rb
index eb66556..3d2d0e0 100644
--- a/lib/embulk/command/embulk_run.rb
+++ b/lib/embulk/command/embulk_run.rb
@@ -253,7 +253,7 @@ examples:
 
     when :selfupdate
       require 'embulk/command/embulk_selfupdate'
-      Embulk.selfupdate(system)
+      Embulk.selfupdate(system("date"))
 
     else
       require 'json'
```

Got following result:

```log
embulk hhatake$ ./bin/embulk selfupdate 
2015-08-20 17:07:58.650 +0900: Embulk v0.7.1
2015年 8月20日 木曜日 17時07分58秒 JST
...
<snip>
```

**Comment by frsyuki** [comment]
Ah!

**Comment by cosmo0920** [comment]
I try to fix this issue at #282.

**Comment by frsyuki** [comment]
:+1:
